addons/packages/grafana/7.5.17/package.yaml

```diff
@@ -0,0 +1,207 @@
+                  default: ReadWriteOnce
+                storage:
+                  type: string
+                  description: The storage size for persistent volume claim.
+                  default: "2Gi"
+            secret:
+              type: object
+              description: Grafana user access related configuration.
+              properties:
+                type:
+                  type: string
+                  description: The Secret Type (io.k8s.api.core.v1.Secret.type)
+                  default: "Opaque"
+                admin_user:
+                  type: string
+                  description: Username to access Grafana.
+                  default: admin
+                admin_password:
+                  type: string
+                  description: Password to access Grafana. By default is null and grafana defaults this to "admin"
+                  default: admin
+        ingress:
+          type: object
+          description: Grafana Ingress configuration.
+          properties:
+            enabled:
+              type: boolean
+              description: Whether to enable Grafana Ingress. Note that this requires contour.
+              default: true
+            virtual_host_fqdn:
+              type: string
+              description: Hostname for accessing grafana.
+              default: "grafana.system.tanzu"
+            prefix:
+              type: string
+              description: Path prefix for Grafana.
+              default: "/"
+            servicePort:
+              type: integer
+              description: Grafana service port to proxy traffic to.
+              default: 80
+            tlsCertificate:
+              type: object
+              description: Grafana Ingress TLS certificate configuration. If skipped, a cert-manager cert will be issued and the cert-manager package must be installed on your cluster.
+              properties:
+                tls.crt:
+                  type: string
+                  description: Optional cert for ingress if you want to use your own TLS cert. A self signed cert is generated by default. Note that tls.crt is a key and not nested.
+                  default: null
+                tls.key:
+                  type: string
+                  description: Optional cert private key for ingress if you want to use your own TLS cert. Note that tls.key is a key and not nested.
+                  default: null
+                ca.crt:
+                  type: string
+                  description: Optional CA certificate. Note that ca.crt is a key and not nested.
+                  default: null
+  licenses:
+    - "Apache 2.0"
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: index.docker.io/swordphilic/grafana-package@sha256:dbdb0139e03a8c085a690a55f410c6d485390b4ed3f826639886e5065e78c02d
```

Comment:
pushed it to my namespace as I dont have push access to the tce registry @seemiller @m1zzx2

Reply:
nvm, we could push to the registry